references/cli/adopt.go

```diff
@@ -143,8 +146,12 @@ func (opt *AdoptOptions) parseResourceRef(f velacmd.Factory, cmd *cobra.Command,
 	case 3:
 		or.Namespace = parts[1]
 		or.Name = parts[2]
+	case 4:
+		or.Cluster = parts[1]
+		or.Namespace = parts[2]
+		or.Name = parts[3]
 	default:
-		return nil, fmt.Errorf("resource should be like <type>/<name> or <type>/<namespace>/<name>")
+		return nil, fmt.Errorf("resource should be like <type>/<name> or <type>/<namespace>/<name> or <type>/<cluster>/<namespace>/<name>")
```

Comment:
I think <cluster> should be first argument. The order should be `<cluster>/<type>/<namespace>/<name>`

Reply:
All right, the origin order comes from description in #5464 , i will change the order to `<cluster>/<type>/<namespace>/<name>`

Reply:
I would prefer <type>/<cluster>/<namespace>/<name> as <cluster>/<namespace>/<name> as a whole could be treated as a complete resource identifier.